media-libs/openjpeg/openjpeg-2.1.2.recipe

```diff
@@ -20,18 +20,18 @@ LICENSE="BSD (2-clause)"
 REVISION="1"
 SOURCE_URI="http://github.com/uclouvain/openjpeg/archive/v$portVersion.tar.gz"
 SOURCE_FILENAME="openjpeg-$portVersion.tar.gz"
-CHECKSUM_SHA256="82c27f47fc7219e2ed5537ac69545bf15ed8c6ba8e6e1e529f89f7356506dbaa"
+CHECKSUM_SHA256="4ce77b6ef538ef090d9bde1d5eeff8b3069ab56c4906f083475517c2c023dfa7"
 PATCHES="openjpeg-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
 
 PROVIDES="
 	openjpeg$secondaryArchSuffix = $portVersion
-	cmd:opj_decompress = $portVersion
-	cmd:opj_compress = $portVersion
-	cmd:opj_dump = $portVersion
-	lib:libopenjp2$secondaryArchSuffix = 7.0.1 compat >= 7
+	cmd:opj_decompress$secondaryArchSuffix = $portVersion
+	cmd:opj_compress$secondaryArchSuffix = $portVersion
+	cmd:opj_dump$secondaryArchSuffix = $portVersion
+	lib:libopenjp2$secondaryArchSuffix = $portVersion compat >= 7
```

Comment:
@korli: Thanks again for your review and sorry for this late reply...

I don't know what to use instead of **`7.0.1`**. Should I use **`7.0.2`** or **`7.1.0`**. Or should I use something else?

Hmm, the only recipes that need libopenjp2 are documentviewer, pillow and mupdf, and none of them require a specific version of libopenjp2. So maybe we could use **`$portVersion`** without the **`compat >= 7`** or with a **`compat >= 2`**.

Anyway, since I'm not sure of what should be used here, just tell me what's the best option and I'll update the recipe with your suggestion. Thanks!